### PR TITLE
N20: Fishgame and nakama-rs

### PR DESCRIPTION
**Pull request by not-fl3**
#531

### CODE REVIEW

File: content/posts/newsletter-020/index.md [hunk 1]
```diff
@@ -665,6 +686,21 @@ _Discussions: [/r/rust][planck_reddit], Discord: jojolepro#8057_
...
+### [nakama-rs]
+
+[nakama-rs] is a pure rust implementation of [Nakama] protocol.
+
+[Nakama] is an open-source server designed to power modern games and apps.
+Features include user accounts, chat, social, matchmaker, realtime multiplayer,
+and much [more][heroiclabs].
+
+Being pure rust, [nakama-rs] brings the full API and socket options to any
+platform rust work on.
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-020/index.md:698]:
```suggestion
[nakama-rs] is a pure Rust implementation of the [Nakama] protocol.

[Nakama] is an open-source server designed to power modern games and apps.
Features include user accounts, chat, social, matchmaker, realtime multiplayer,
and much [more][heroiclabs].

Being pure Rust, [nakama-rs] brings the full API and socket options to any
platform Rust works on.
```

**Review by 17cupsofcoffee — APPROVED** [review]

File: content/posts/newsletter-020/index.md [hunk 2]
```diff
@@ -207,6 +207,27 @@ of the past 6 months of development.
 [egregoria-discord]: https://discord.gg/CAaZhUJ
 [egregoria-youtube]: https://youtu.be/qH2SKWbRV5I
 
+### [Fishgame][fishgame]
+
+![Fishgame](fishgame.gif)
```
Comment (ozkriff) [inline content/posts/newsletter-020/index.md:212]: There's no GIF in the PR :)

**Review by ozkriff — CHANGES_REQUESTED** [review]
lgtm overall except for a missing GIF

**Comment by not-fl3** [comment]
Fixed missing GIF :)

**Review by ozkriff — APPROVED** [review]

**Comment by ozkriff** [comment]
Merged, thanks for the PR!

**Pull request merged**